x/bank/keeper/msg_server.go

```diff
@@ -124,3 +124,30 @@ func (k msgServer) UpdateParams(goCtx context.Context, req *types.MsgUpdateParam
 
 	return &types.MsgUpdateParamsResponse{}, nil
 }
+
+func (k msgServer) SetSendEnabled(goCtx context.Context, msg *types.MsgSetSendEnabled) (*types.MsgSetSendEnabledResponse, error) {
```

Comment:
I think this api is somewhat cleaner than the UpdateParam change from above. Should we decide to only keep one of them cc @alexanderbez

Reply:
Oh right! I had added the ability to set the default here in case it's something that needed to be done at the same time as some specific entries. Plus, it would then get voted on as an all-together thing, similar to how it was previously done in params.

Reply:
cc @alexanderbez

Reply:
Trying to follow along here. Why do you think `SetSendEnabled` is cleaner than `UpdateParams`?

Reply:
They both share the same logic basically, ideally we have one way of doing these modifications instead of two

Reply:
Both `UpdateParams` and `SetSendEnabled` can update the `DefaultSendEnabled`. But `SetSendEnabled` can also set/delete specific denom `SendEnabled` flags.

I included the `DefaultSendEnabled` stuff in `SetSendEnabled` since that default is intimately related to specific denom `SendEnabled` flags.

If we only want a single way to set `DefaultSendEnabled`, I think removing `UpdateParams` is better than removing the ability to set `DefaultSendEnabled` from `SetSendEnabled`. I find it likely that, when `DefaultSendEnabled` needs to be updated, some specific denom `SendEnabled` entries will too. By having them together, that can be done in a single proposal. The only downside I see to removing `UpdateParams` is that it's an expected endpoint for a module that uses `Params`, which might cause some confusion.

Reply:
Actually. I just saw the change in `x/gov` (as of 0.48) to allow for multiple messages in a governance proposal. So, I'll remove the `DefaultSendEnabled` stuff from the `SetSendEnabled` endpoint.